db/migrate/20220106122727_create_reminders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReminders < ActiveRecord::Migration[6.0]
+  def change
+    create_table :reminders do |t|
+      t.datetime :due_at, null: false
```

Comment:
```suggestion
      t.datetime :due_at, null: false, index: true
```
Porque seguro después lo usamos para buscar y notificar.

Reply:
perfecto